### PR TITLE
[MM-485] - Update `commonHelpText` and `sysAdminHelpText` to include all slash commands

### PR DESCRIPTION
**Pull request by Francois-D**
#### Summary

*Problem*
jiraCommandHandler contains the mappings of slash commands to their corresponding command handler functions. The following slash commands are missing and should be added to the HelpText variables.

*Resolution*
Mapped the following commands with their descriptions to the handler functions. 
webhook
stats
info
help

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-plugin-jira/issues/485

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -21,6 +21,10 @@ const commonHelpText = "\n* `/jira connect` - Connect your Mattermost account to
 	"* `/jira unassign <issue-key>` - Unassign the Jira issue\n" +
 	"* `/jira create <text (optional)>` - Create a new Issue with 'text' inserted into the description field\n" +
 	"* `/jira transition <issue-key> <state>` - Change the state of a Jira issue\n" +
+	"* `/jira stats` - Poll basic statistics about the Jira plug-in.  Recevie performace metrics and authenticated users between Jira/Mattermost\n" +
```
Comment (jfrerich) [inline server/command.go:24]:
```suggestion
	"* `/jira stats` - Poll basic statistics about the Jira plug-in.  Receive performance metrics and authenticated users between Jira/Mattermost\n" +
```

File: server/command.go [hunk 2]
```diff
@@ -21,6 +21,10 @@ const commonHelpText = "\n* `/jira connect` - Connect your Mattermost account to
...
+	"* `/jira info` - Retrieve information about the current user and the Jira plug-in\n" +
+	"* `/jira help` - Launch the Jira plug-in command line help syntax\n" +
```
Comment (jfrerich) [inline server/command.go:26]:
```suggestion
	"* `/jira help` - Launch the Jira plugin command line help syntax\n" +
```

File: server/command.go [hunk 3]
```diff
@@ -21,6 +21,10 @@ const commonHelpText = "\n* `/jira connect` - Connect your Mattermost account to
...
+	"* `/jira webhook` - Execute the Jira plug-in webhook. The Jira plug-in gets sent a stream of pre-configured events from the Jira server\n" +
```
Comment (jfrerich) [inline server/command.go:27]:
```suggestion
	"* `/jira webhook` - Execute the Jira plug-in webhook. The Jira plugin gets sent a stream of pre-configured events from the Jira server\n" +
```

**Comment by Francois-D** [comment]
@jfrerich sorry about the typos.
I've commited my edits.

File: server/command.go [hunk 4]
```diff
@@ -698,7 +702,7 @@ func getCommand() *model.Command {
 		DisplayName:      "Jira",
 		Description:      "Integration with Jira.",
 		AutoComplete:     true,
-		AutoCompleteDesc: "Available commands: connect, assign, disconnect, create, transition, view, subscribe, settings, install cloud/server, uninstall cloud/server, help",
+		AutoCompleteDesc: "Available commands: connect, assign, disconnect, create, transition, stats, info, webhook,  view, subscribe, settings, install cloud/server, uninstall cloud/server, help",
```
Comment (levb) [inline server/command.go:705]: remove stats

File: server/command.go [hunk 5]
```diff
@@ -21,6 +21,10 @@ const commonHelpText = "\n* `/jira connect` - Connect your Mattermost account to
 	"* `/jira unassign <issue-key>` - Unassign the Jira issue\n" +
 	"* `/jira create <text (optional)>` - Create a new Issue with 'text' inserted into the description field\n" +
 	"* `/jira transition <issue-key> <state>` - Change the state of a Jira issue\n" +
+	"* `/jira stats` - Poll basic statistics about the Jira plug-in.  Receive performance metrics and authenticated users between Jira/Mattermost\n" +
+	"* `/jira info` - Retrieve information about the current user and the Jira plug-in\n" +
+	"* `/jira help` - Launch the Jira plugin command line help syntax\n" +
+	"* `/jira webhook` - Execute the Jira plug-in webhook. The Jira plugin gets sent a stream of pre-configured events from the Jira server\n" +
```
[the remark this answers is not in the document]
Comment (Francois-D) [reply server/command.go:27]: Thanks for the feedback.  The Jira stat description was derived from this location.  
https://mattermost.gitbook.io/plugin-jira/admininstrator-guide/administrator-slash-commands#retrieving-basic-stats

Is this the same slash command that Jira stats is referring to?  
Thanks in advance

File: server/command.go (hunk 4 above)
Comment (Francois-D) [reply server/command.go:705]: The following commands are also listed as system administrator commands should they be removed as well including 'stats'?
_Install cloud/server, webhook, subscribe._

Comment (levb) [reply server/command.go:705]: Yes, thanks for catching these.

Comment (Francois-D) [reply server/command.go]: Thanks I've made the edits. :)

File: server/command.go [hunk 6]
```diff
@@ -21,6 +21,10 @@ const commonHelpText = "\n* `/jira connect` - Connect your Mattermost account to
 	"* `/jira unassign <issue-key>` - Unassign the Jira issue\n" +
 	"* `/jira create <text (optional)>` - Create a new Issue with 'text' inserted into the description field\n" +
 	"* `/jira transition <issue-key> <state>` - Change the state of a Jira issue\n" +
+	"* `/jira stats` - Retrieve version information about the Jira plugin\n" +
```
Comment (levb) [inline server/command.go:24]: 2/5 suggest: "Display usage statistics"

File: server/command.go [hunk 7]
```diff
@@ -21,6 +21,10 @@ const commonHelpText = "\n* `/jira connect` - Connect your Mattermost account to
...
+	"* `/jira info` - Retrieve information about the current user and the Jira plug-in\n" +
```
Comment (levb) [inline server/command.go:25]: 1/5 `s/Retrieve/Display/`

File: server/command.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (levb) [reply server/command.go:27]: `/jira webhook` should be "Display the URL to configure Jira webhooks to".

File: server/command.go (hunk 4 above)
Comment (levb) [reply server/command.go]: @Francois-D also, these descriptions belong in the SysAdmin-specific section, just a few lines below

**Comment by levb** [comment]
@Francois-D also, these descriptions belong in the SysAdmin-specific section, just a few lines below

File: server/command.go (hunk 6 above)
Comment (Francois-D) [reply server/command.go]: Thanks.
I've updated description.

**Comment by Francois-D** [comment]
> @Francois-D also, these descriptions belong in the SysAdmin-specific section, just a few lines below

Thanks I've moved the command to the correct area.  I also updated the subscribe list description with the official one from the documentation from the link 

https://mattermost.gitbook.io/plugin-jira/admininstrator-guide/administrator-slash-commands#jira-subscribe-list-command

**Comment by DHaussermann** [comment]
@Francois-D Thanks for updating this!
One small point about some of the new text - for `/jira webhook` the new text is "_Configure the Mattermost webhook to receive JQL queries_" 
As far as I know, there is no way to change any of the configuration from this command. Would it maybe make more sense to just have `/jira webhook` the new text is "_**Show** the Mattermost webhook to receive JQL queries_"

**Comment by Francois-D** [comment]
@DHaussermann 
Thanks I've updated the WebHooks definition.

**Comment by Francois-D** [comment]
@DHaussermann 
May I suggest informing the user to run /jira webhook? This would aid in setting up the plugin.

Screen Shot 2020-04-28 at 1 49 04 PM


Example of running /jira webhook
Screen Shot 2020-04-28 at 1 56 40 PM

**Pull request merged**